pkgs/games/qgo/default.nix

```diff
@@ -43,6 +43,6 @@ stdenv.mkDerivation rec {
     # libQt5XcbQpa is a platform plugin dependency and doesn't get linked
     patchelf --add-needed libQt5XcbQpa.so.5 $out/bin/qgo
     wrapProgram $out/bin/qgo \
-      --set QT_QPA_PLATFORM_PLUGIN_PATH "${qt56.qtbase}/lib/qt-5.6/plugins/platforms/"
+      --set QT_QPA_PLATFORM_PLUGIN_PATH "${stdenv.lib.getLib qt56.qtbase}/lib/qt-5.6/plugins/platforms/"
```

Comment:
This should be `${stdenv.lib.getBin qt56.qtbase}/$qtPluginPrefix/platforms/`. The plugins are installed in `bin` because they are not exactly libraries and they need to be installed alongside any executables anyway.

Reply:
Thanks, fixed & amended.